docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -129,6 +129,29 @@ In Sourcegraph 5.2 and earlier, you should use the feature flag `cody` to turn C
 />
 
 
+## Use StarCoder for Autocomplete
+
+When tested with other coder models for the autocomplete use case, [StarCoder](https://huggingface.co/blog/starcoder) offered significant improvements in quality and latency compared to our control groups for users on Sourcegraph.com. You can read more about the improvements in our [October 2023 release notes](https://sourcegraph.com/blog/feature-release-october-2023) and the [GA release notes](https://sourcegraph.com/blog/cody-is-generally-available).
+
+Since Sourcegraph 5.3, every Cody Enterprise setup that uses [Cody Gateway](/cody/core-concepts/cody-gateway) can opt-in to using StarCoder as an alternative to Claude Instant for autocomplete. To ensure a fast and reliable experience, we are partnering with [Fireworks](https://fireworks.ai/) and have set up a dedicated dedicated hardware deployment for our Enterprise users.
+
+To enable StarCoder go to **Site admin > Site configuration** (`/site-admin/configuration`) and change the `completionModel`:
```

Comment:
@eshamow @michaellzc would we expect the Cloud customers to make this change manually? Or, would they ask us (Sourcegraph) to apply the change to a given Cloud instance?